.github/workflows/test.yml

```diff
@@ -149,29 +149,35 @@ jobs:
           use-mamba: true
           python-version: "3.8"
           channel-priority: strict
+          channels: ${{ needs.detect-ci-trigger.outputs.triggered == 'true' && 'dask/label/dev,conda-forge,nodefaults' || 'conda-forge,nodefaults' }}
           activate-environment: dask-sql
           environment-file: continuous_integration/environment-3.8-jdk11-dev.yaml
       - name: Download the pre-build jar
         uses: actions/download-artifact@v1
         with:
           name: jar
           path: dask_sql/jar/
-      - name: Install dependencies
+      - name: Install cluster dependencies
         run: |
           mamba install python-blosc lz4 -c conda-forge
 
           which python
           pip list
           mamba list
-      - name: Optionally install upstream dev Dask / dask-ml
+      - name: Optionally install upstream dev dask-ml
         if: needs.detect-ci-trigger.outputs.triggered == 'true'
         run: |
-          python -m pip install --no-deps git+https://github.com/dask/dask
-          python -m pip install --no-deps git+https://github.com/dask/distributed
+          mamba update dask
           python -m pip install --no-deps git+https://github.com/dask/dask-ml
```

Comment:
Maybe we should add a `dask-ml` nightly next 🙂

Reply:
Yeah was thinking about that 🙂 happy to do the work if Dask maintainers are okay with it, can open up an issue around this

Reply:
Maybe raise an issue on the Dask-ML repo asking this question. Can also link back to issue ( https://github.com/dask/community/issues/76 ) where we had this discussion about Dask & Distributed